### ISSUE TITLE
Dropbox sync broken

### ISSUE THREAD

**Issue by roniziv** [MiniKeePass/MiniKeePass#508]
Since upgrading to iOS 11 dropbox sync is broken. Please fix it.
Please also check if iOS 11 supports seamless sync with dropbox via the new files app/api

**Comment by alie2n** [comment]
I would second the seamless Dropbox sync. That would be much easier. 
That way ownCloud/Nextcloud could be supported also.

**Comment by dmgm** [comment]
This is the reason I have Keepass Touch installed. Much easier two way dropbox support.
It's the missing feature for me. Saving to dropbox from minikeypass is easy, but if you make a change from a desktop client you need to re-export open with minikeepass.

**Comment by 91DarioDev** [comment]
@dmgm yes you're right but Keepass Touch, despite the developer says it's opensource, according to my experience it isn't.

The code is not hosted on github, but you have to send an email to the developer to have the code.
i sent a couple of emails some months ago and still no answers. so i would not consider Keepass Touch opensource.

**Comment by dmgm** [comment]
Yes. I favour open source apps, so Keepass is somewhat deceptive in advertising as open source, but not providing the source. I don't see that they are using an open source license either.

Ideally the minikeepass author would in incorporate some sort of dropbox/file opening mechanism other than the current export approach, and I could then delete keepass touch.

**Comment by 91DarioDev** [comment]
@dmgm i miss the feature too very much. btw i choose keypass for being opensource, so i won't use a closed source mobile client for any reason. otherwise i would have chosen a real cloud password manager.

also: keepass touch miss an important feature that is present on the android client: ability to give the app access to dropbox only at one folder and not the whole storage.

**Comment by Nottt** [comment]
So no plans on implementing two-way sync at all? Any reason it lacks such a basic feature?

**Comment by roniziv** [comment]
BTW, KeePass Touch supports such functionality.

**Comment by dmgm** [comment]
I admit I am using keepass touch because of this functionality. I would prefer to use minikeepass because it's truely open source. That said, it's free software the result of the authors coding. I total respect his (or her) decision to code whatever pleases them.
